pkg/common-controller/snapshot_controller.go

```diff
@@ -405,6 +405,9 @@ func (ctrl *csiSnapshotCommonController) syncUnreadySnapshot(snapshot *crdv1.Vol
 			return err
 		}
 
+		msg := fmt.Sprintf("waiting for a snapshot %s to be created", snapshot.Name)
```

Comment:
We don't need this event for pre-provisioned snapshot.

Reply:
copy that